src/commonTest/kotlin/fr/acinq/lightning/channel/InteractiveTxTestsCommon.kt

```diff
@@ -1213,10 +1215,15 @@ class InteractiveTxTestsCommon : LightningTestSuite() {
             return action1
         }
 
-        private fun createWallet(onChainKeys: KeyManager.SwapInOnChainKeys, amounts: List<Satoshi>): List<WalletState.Utxo> {
+        private fun createWallet(onChainKeys: KeyManager.SwapInOnChainKeys, amounts: List<Satoshi>, legacyAmounts: List<Satoshi> = listOf()): List<WalletState.Utxo> {
```

Comment:
You're introducing these two new parameters but you don't even use them once...you should modify at least one of the existing tests to provide both taproot inputs and legacy inputs, otherwise we don't have any test coverage of the legacy swap protocol anymore.